src/main/java/org/candlepin/controller/Entitler.java

```diff
@@ -301,8 +300,7 @@ public List<Entitlement> bindByProducts(AutobindData data, boolean force)
 
                 throw new ForbiddenException(i18n.tr(
                     "Development units may only be used on hosted servers" +
-                    " and with orgs that have active subscriptions."
-                ));
+                        " and with orgs that have active subscriptions."));
```

Comment:
Done. Reformatted to remove indent.

Note for the following indent comments. Most of them are not unnecessary. They are line continuations. Indent signifies that it is a continuation of a single line rather than a new line of the same level.

Reply:
Yeah, and in many cases it's fine. But it's when we indent off a continuation that spans multiple lines, we get multiple indentations instead of just a single indentation, as is the case here.